server/database/fakeData/data.json

```diff
@@ -0,0 +1,201 @@
+{
+  "campaigns": [
+    {
```

Comment:
This is better than before, but I need you to create a folder for your fake data and add `json` file to each table separately.